src/core/pdf-parser/parseNumber.ts

```diff
@@ -19,11 +19,12 @@ const parseNumber = (
   { onParseNumber }: IParseHandlers = {},
 ): [PDFNumber, Uint8Array] | void => {
   const trimmed = trimArray(input);
-  const numRegex = /^(((\+{1}|-{1})?\d+(\.\d+)?)|((\+{1}|-{1})?\.\d+))/;
+  const numRegex = /^(([+-]?\d+(\.\d+)?)|([+-]?\.\d+))/;
 
   // Search for the first character that isn't part of a number
   let idx = 0;
   while (String.fromCharCode(trimmed[idx]).match(/^[+-.\d]/)) idx += 1;
+  console.log('parseNumber', idx, arrayToString(trimmed, 0, 100));
```

Comment:
```suggestion
```
I assume this was just for debugging purposes?

Reply:
Yes, sorry, shouldn't have been committed.